### PR TITLE
Update changelog to add new 'relative links' feature

### PR DESCRIPTION
**Pull request by lfdebrux**
## What’s changed

Updates changelog to include new features from PR #291.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,23 @@
...
+set :relative_links, true
+activate :relative_assets
+```
+
+This may help with deploying to GitHub pages.
+
+- [#291: Support sites deployed on paths other than "/" (by generating relative links)](https://github.com/alphagov/tech-docs-gem/pull/291)
+
+Thanks [@eddgrant](https://github.com/eddgrant) for contributing this feature and the associated fixes!
+
```
Comment (PippaClarkGDS) [inline CHANGELOG.md:21]: ## 3.2.0

### New features 

You can now [configure your Tech Docs Template (TDT) to build your documentation site to use relative links to pages and assets](https://tdt-documentation.london.cloudapps.digital/configure_project/global_configuration/#build-your-site-using-relative-links).

Thanks [@eddgrant](https://github.com/eddgrant) for contributing this feature and the associated fixes.

This change was introduced in [pull request #291: Support sites deployed on paths other than "/" (by generating relative links)](https://github.com/alphagov/tech-docs-gem/pull/291).

**Review by PippaClarkGDS — COMMENTED** [review]
have inputted the release note text

File: CHANGELOG.md (hunk 1 above)
Comment (lfdebrux) [reply CHANGELOG.md:21]: Yep!

Comment (PippaClarkGDS) [reply CHANGELOG.md:21]: awesome - do you need me to do anything or are you ok to commit it in? I can review

Comment (lfdebrux) [reply CHANGELOG.md:21]: I'll commit it in

Comment (lfdebrux) [reply CHANGELOG.md]: Done!

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,14 @@
 
 ## Unreleased
 
+### New features
```
Comment (PippaClarkGDS) [inline CHANGELOG.md:5]:
```suggestion
## 3.2.0

### New features
```

**Review by PippaClarkGDS — COMMENTED** [review]
small tweak to include the release note number, otherwise looks good

File: CHANGELOG.md (hunk 2 above)
Comment (lfdebrux) [reply CHANGELOG.md:5]: Oh yeah, I was gonna add in that in a subsequent release PR, just makes more sense to me that way.

Comment (PippaClarkGDS) [reply CHANGELOG.md:5]: ah great - is that the usual way it's done? If so, I'll approve it

Comment (lfdebrux) [reply CHANGELOG.md:5]: Yeah, at least on other projects I've worked on.

**Review by PippaClarkGDS — APPROVED** [review]
Approved without the numbering as this will be in the release PR

**Pull request merged**